demisto_sdk/commands/common/constants.py

```diff
@@ -781,7 +781,7 @@ class PB_Status:
 INTEGRATION_CATEGORIES = ['Analytics & SIEM', 'Utilities', 'Messaging', 'Endpoint', 'Network Security',
                           'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis',
                           'IT Services', 'Data Enrichment & Threat Intelligence', 'Authentication', 'Database',
-                          'Deception', 'Email Gateway', 'Identity and Access Management']
+                          'Deception', 'Email Gateway', 'Identity and Access Management', 'File Integrity Management']
```

Comment:
This will this let `init` create this integration type right?

Reply:
right